opendevin/config.py

```diff
@@ -5,12 +5,29 @@
 
 load_dotenv()
 
+DEFAULT_CONFIG = {
+    "LLM_API_KEY": None,
+    "LLM_BASE_URL": None,
+    "WORKSPACE_DIR": os.path.join(os.getcwd(), "workspace"),
+    "LLM_MODEL": "gpt-4-0125-preview",
+    "SANDBOX_CONTAINER_IMAGE": "ghcr.io/opendevin/sandbox",
+    "RUN_AS_DEVIN": "false",
+    "LLM_EMBEDDING_MODEL": "local",
+    "LLM_NUM_RETRIES": 6,
+    "LLM_COOLDOWN_TIME" : 1,
+    "DIRECTORY_REWRITE" : "",
+    "PROMPT_DEBUG_DIR": "",
+}
+
 config_str = ""
 if os.path.exists("config.toml"):
     with open("config.toml", "rb") as f:
         config_str = f.read().decode("utf-8")
 
 config = toml.loads(config_str)
+for key, value in DEFAULT_CONFIG.items():
+    if key not in config:
+        config[key] = value if value is not None else os.environ.get(key)
```

Comment:
It might be nice to invert this logic, so that people can't rely on settings that aren't listed in DEFAULT_CONFIG.

I.e.
```
config = DEFAULT_CONFIG.copy()
for key, value in config.items():
  if key in os.environ:
    config[key] = os.environ[key]
  elif key in tomlConfig:
    config[key] = tomlConfig[key]
```

Reply:
Good point. Done.